Content.Server/_Impstation/Radio/SelfHeadsetComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Robust.Shared.Prototypes;
+using Content.Shared.Radio;
+
+namespace Content.Server._Impstation.Radio.Components;
```

Comment:
this isn't in a `/Components` folder, so you can just cut  out the `.Components` at the end here